rules/declaration-strict-value.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+// See https://github.com/AndyOGo/stylelint-declaration-strict-value
+
+const properties = [
+    '/color/',
+    'font-family',
+    'font-weight',
+    'z-index',
+];
+
+module.exports = {
+    'plugins': [
+        'stylelint-declaration-strict-value',
+    ],
+    'rules': {
+        'scale-unlimited/declaration-strict-value': [properties, {}],
```

Comment:
we need to ignore the enforce of the variables in certain cases like `currentColor`, `transparent`, `inherit`.

Reply:
Added an option to ignore `currentColor`, `transparent` and `inherit`.